modules/account-lib/test/unit/coin/dot/transactionBuilder/unnominateBuilder.ts

```diff
@@ -0,0 +1,103 @@
+import should from 'should';
+import { UnnominateBuilder } from '../../../../../src/coin/dot';
+import * as DotResources from '../../../../resources/dot';
+import { buildTestConfig } from './base';
+
+describe('Dot Unnominate Builder', () => {
+  let builder: UnnominateBuilder;
+
+  const sender = DotResources.accounts.account1;
+
+  beforeEach(() => {
+    const config = buildTestConfig();
+    builder = new UnnominateBuilder(config);
+  });
+
+  describe('build unnominate transaction', () => {
+    it('should build a unnominate transaction', async () => {
+      builder
+        .sender({ address: sender.address })
+        .validity({ firstValid: 3933, maxDuration: 64 })
+        .referenceBlock('0x149799bc9602cb5cf201f3425fb8d253b2d4e61fc119dcab3249f307f594754d')
+        .sequenceId({ name: 'Nonce', keyword: 'nonce', value: 200 })
+        .fee({ amount: 0, type: 'tip' })
+        .version(8);
+      builder.sign({ key: sender.secretKey });
+      const tx = await builder.build();
+      const txJson = tx.toJson();
+      should.deepEqual(txJson.sender, sender.address);
+      should.deepEqual(txJson.blockNumber, 3933);
+      should.deepEqual(txJson.referenceBlock, '0x149799bc9602cb5cf201f3425fb8d253b2d4e61fc119dcab3249f307f594754d');
+      should.deepEqual(txJson.genesisHash, '0xe143f23803ac50e8f6f8e62695d1ce9e4e1d68aa36c1cd2cfd15340213f3423e');
+      should.deepEqual(txJson.specVersion, 9150);
```

Comment:
I think we should start initializing a variable for `specVersion` at the start of the unit tests for each builder. We've ha to go back and manually change each specVersion every time there is a node update, so it would make things easier just to change 1 value per builder's test set. Same goes for genesisHash and referenceBlock I think - maybe we should use variables instead of hard coding them into the unit tests.

Reply:
I set variables for those 3 values like you suggested.